cargo-process.el

```diff
@@ -178,8 +178,7 @@
   "Subcommand used by `cargo-process-check'."
   :type 'string)
 
-(defcustom cargo-process--command-clippy "clippy -Zunstable-options"
-  "Subcommand used by `cargo-process-clippy'. Uses `-Zunstable-options` to work around https://github.com/rust-lang/rust-clippy/issues/4612."
```

Comment:
Could you re-add the `"Subcommand used by `cargo-process-clippy'."` part of the docstring?

Reply:
Whoops, missed that. I pushed a new commit.  PTAL.